src/main/java/Event.java

```diff
@@ -0,0 +1,75 @@
+import java.time.DateTimeException;
+import java.time.LocalDateTime;
+
+public class Event extends Task {
+    private LocalDateTime startDateTime;   // DD/MM/YYYY
+    private LocalDateTime endDateTime;
+
+    public Event(String description, String timePeriod) throws DukeException{
+        super(description);
+        this.setDateTime(timePeriod);
+    }
+
+    private void setDateTime(String timePeriod) throws DukeException{
```

Comment:
Perhaps it would be a good idea to do the parsing before this (in `Parser` for instance). This way you wouldn't have duplicate code and it saves you some work :D

Reply:
ah. I'll look into it thanks!